.pre-commit-config.yaml

```diff
@@ -9,9 +12,9 @@ repos:
     hooks:
     - id: nbqa-black
     - id: nbqa-isort
-      args: [ --nbqa-mutate, --profile=black, --project=pandas_profiling ]
+      args: [ --profile=black, --project=pandas_profiling ]
```

Comment:
why `nbqa-mutate` was dropped?

Reply:
while testing the new approach, I noted a warning during execution stating this optional flag has been dropped since it was now the default behavior. I can revert this change if you fell more comfortable